site/content/en/docs/administration/advanced/upgrade_guide.md

```diff
@@ -56,6 +56,13 @@ To upgrade CVAT, follow these steps:
   docker logs cvat_server -f
   ```
 
+## How to upgrade CVAT from v2.45.0 to v2.46.0.
+
+In version 2.46.0, CVAT upgraded the FFMPEG library it uses to split videos into frames from 4.3.1 to 8.0.
+There is a small chance that some video files may not be processed correctly with the new FFMPEG version.
```

Comment:
I think "correctly" is not really the concern here - it could very much be that a video _is_ processed correctly, but it wasn't before. The concern is only that it could be processed differently from the old version.